src/search.js

```diff
@@ -44,23 +44,18 @@ Search for packages/themes.\
         json: true
       };
 
-      return new Promise((resolve, reject) => {
-        request.get(requestSettings, function(error, response, body) {
-          body ??= {};
-          if (error != null) {
-            return void reject(error);
-          }
-          if (response.statusCode === 200) {
-            let packages = body.filter(pack => (pack.releases != null ? pack.releases.latest : undefined) != null);
-            packages = packages.map(({readme, metadata, downloads, stargazers_count}) => _.extend({}, metadata, {readme, downloads, stargazers_count}));
-            packages = packages.filter(({name, version}) => !isDeprecatedPackage(name, version));
-            return void resolve(packages);
-          }
-
-          const message = request.getErrorMessage(body, error);
-          reject(`Searching packages failed: ${message}`);
-        });
-      });
+      const response = await request.get(requestSettings);
+      const body = response.body ?? {};
+
+      if (response.statusCode === 200) {
+        let packages = body.filter(pack => pack?.releases?.latest != null);
+        packages = packages.map(({readme, metadata, downloads, stargazers_count}) => _.extend({}, metadata, {readme, downloads, stargazers_count}));
+        packages = packages.filter(({name, version}) => !isDeprecatedPackage(name, version));
+        return packages;
+      }
+
+      const message = request.getErrorMessage(body, error);
```

Comment:
The second argument `error` here is never defined. Is it intended to be collected in a missing `try...catch` statement?

Reply:
I don't know, I didn't create `getErrorMessage` to begin with. 😅

Reply:
Well I just mean, your changes removing the definition of the variable `error`. So I was asking if you intended to have it defined elsewhere, but if not then we will want to instead do `request.getErrorMessage(body, null);`